ports/drilbertiidiggeredoo/port.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": 2,
+    "name": "drilbertiidiggeredoo.zip",
+    "items": [
+        "Drilbert II Diggeredoo.sh",
+        "drilbertiidiggeredoo/"
```

Comment:
/ after the portname in items

Reply:
That's how it is supposed to be?

Reply:
Will wait for Cebion and Kloptops to decide who is right :)

Reply:
> That's how it is supposed to be?

i checked 8 ports or so and not one has / behind it :D

Reply:
Freshly generated from portmaster.games

```
{
  "version": 3,
  "name": "test.zip",
  "items": [
    "Test.sh",
    "test"
  ],
  "items_opt": [],
  "attr": {
    "title": "Testgame",
    "porter": [
      "Cebion"
    ],
    "desc": "asd",
    "desc_md": "ads",
    "inst": "asd",
    "inst_md": "ads",
    "genres": [
      "adventure"
    ],
    "image": null,
    "rtr": false,
    "exp": false,
    "runtime": null,
    "reqs": [],
    "arch": [
      "x86_64"
    ]
  }
}```

Reply:
Cool -- removed the trailing slash